features/fixtures/all_content.json

```diff
@@ -141,30 +141,30 @@
     "available_translations": [
       {
         "api_path": "/api/content/search/all",
-        "api_url": "https://www.integration.publishing.service.gov.uk/api/content/search/all",
+        "api_url": "https://www.gov.uk/api/content/search/all",
         "base_path": "/search/all",
         "content_id": "dd395436-9b40-41f3-8157-740a453ac972",
         "document_type": "finder",
         "links": {},
         "locale": "en",
-        "public_updated_at": "2024-09-20T09:17:24Z",
+        "public_updated_at": "2024-10-24T11:19:57Z",
```

Comment:
It's odd that the dates needed to be updated.

Reply:
That's just a side effect of importing the live content item – I think it's probably helpful to leave the date changes in even though they don't matter, because that way in the future someone might be able to tell when the fixture was last updated!